.editorconfig

```diff
@@ -24,3 +24,6 @@ insert_final_newline = false
 
 [Makefile]
 indent_style = tab
+
+[jsonnetfile.json]
+insert_final_newline = false
```

Comment:
I know.
Unfortunately `projectsyn/commodore:latest component compile .` always creates the jsonnetfile.json without the trailing newline and even removes it if the file and newline is already there. Commodore also adds a dependency to `https://github.com/bitnami-labs/kube-libsonnet` even though the component itself doesn't need it in any of the jsonnet files.
I could add the file .gitignore since I don't actually need dependencies for this particular component.

Reply:
IMO, if we don't need the dependency, adding `jsonnetfile.json` to `.gitignore` is ok. `commodore compile` generates the file in order to allow legacy components which assume that `kube.libjsonnet` will be made available by Commodore itself to compile

Reply:
Ok, will do that